src/modules/dashboard/components/SmiteDialog/SmiteDialog.tsx

```diff
@@ -58,30 +82,59 @@ const SmiteDialog = ({
       .required()
       .moreThan(0, () => MSG.amountZero),
     annotation: yup.string().max(4000),
+    forceAction: yup.boolean(),
+    motionDomainId: yup.number(),
   });
 
   const { data: colonyMembers } = useMembersSubscription({
     variables: { colonyAddress },
   });
 
+  const transform = useCallback(
+    pipe(
+      mapPayload(({ amount, domainId, annotation, user, motionDomainId }) => {
+        const totalReputation = bigNumberify(totalReputationData || '0');
+        const repuationChangeAmount = totalReputation.mul(amount).div(100);
+
+        return {
+          colonyAddress,
+          colonyName,
+          domainId,
+          userAddress: user.profile.walletAddress,
+          annotationMessage: annotation,
+          amount: `-${repuationChangeAmount.toString()}`,
```

Comment:
I would have written it as:
```suggestion
          amount: String(-1 * repuationChangeAmount),
```
just to be more readable